NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -550,90 +550,94 @@ public async Task<bool> LoadAsync()
     }
 
     /// <summary>
-    /// Gets the income amount for the date range
+    /// Gets the total income amount for the transactions given
     /// </summary>
-    /// <param name="endDate">The end date</param>
-    /// <param name="startDate">The start date</param>
-    /// <returns>The income amount for the date range</returns>
-    public decimal GetIncome(DateOnly endDate, DateOnly? startDate = null)
+    /// <param name="transactionIds">The ids of transactions to consider</param>
+    /// <returns>The income amount</returns>
+    public decimal GetIncome(IEnumerable<uint>? transactionIds = null)
```

Comment:
You can simplify these even more to just:
```csharp
public decimal GetIncome(IEnumerable<uint>? transactionIds = null) => (transactionIds ?? Transactions.Keys)
            .Select(id => Transactions[id])
            .Where(transaction => transaction.Type == TransactionType.Income)
            .Sum(transaction => transaction.Amount);
```
and the other methods as well.

Reply:
Done. Thanks